solver/app/check_internal_test.go

```
@@ -38,6 +39,16 @@ func TestCheck(t *testing.T) {
 	outbox := addrs.SolverNetOutbox
 
 	for _, tt := range checkTestCases(t, solver, outbox) {
+		// TODO(zodomo): Remove this once network upgrade is complete
+		if tt.req.SourceChainID == evmchain.IDOmniMainnet || tt.req.SourceChainID == evmchain.IDOmniOmega || tt.req.SourceChainID == evmchain.IDOmniStaging {
+			continue
+		}
+
+		// TODO(zodomo): Remove this once network upgrade is complete
+		if tt.req.DestinationChainID == evmchain.IDOmniMainnet || tt.req.DestinationChainID == evmchain.IDOmniOmega || tt.req.DestinationChainID == evmchain.IDOmniStaging {
+			continue
+		}
+
```

if you wanna gut check you could over ride expected rejection here, rather than continuing

---

Had to overwrite more than the reason lol, had to overwrite accepted status, set rejected status, and set the rejection reason, then regenerate the golden file for these tests, but then it worked. It's now strict instead of just a bypass.